sphinxcontrib/doxylink/doxylink.py

```diff
@@ -235,37 +257,43 @@ def join(*args):
     return ''.join(args)
 
 
-def create_role(app, tag_filename, rootdir):
+def create_role(app, tag_filename, rootdir, cache_name):
     # Tidy up the root directory path
     if not rootdir.endswith(('/', '\\')):
         rootdir = join(rootdir, os.sep)
 
     try:
-        tag_file = ET.parse(tag_filename)
-
-        cache_name = os.path.basename(tag_filename)
+        if is_url(tag_filename):
+            response = requests.get(tag_filename)
+            if response.status_code != 200:
+                raise FileNotFoundError
```

Comment:
Rather put a sphinx warning for this, as I assume in this way it crashes the build corect?

Reply:
This error gets raised inside of a try-block. It gets caught and an warning about the tag file will be logged.

Reply:
OK, good for me